dotnet/src/SemanticKernel.Abstractions/Filters/Function/FunctionInvokedContext.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Class with data related to function after invocation.
+/// </summary>
+[Experimental("SKEXP0005")]
+public sealed class FunctionInvokedContext : FunctionFilterContext
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="FunctionInvokedContext"/> class.
+    /// </summary>
+    /// <param name="function">The <see cref="KernelFunction"/> with which this filter is associated.</param>
+    /// <param name="arguments">The arguments associated with the operation.</param>
+    /// <param name="result">The result of the function's invocation.</param>
+    public FunctionInvokedContext(KernelFunction function, KernelArguments arguments, FunctionResult result)
```

Comment:
Should we validate that the function result is for the specified function? Or if we can extract the function from the function result, does this even need to take the function separately?

Reply:
All context classes were implemented in exactly the same way as event args. I think if we apply this type of modifications to Context classes, we should probably do the same for event args.

Reply:
Taking into account that function can be accessed from `FunctionResult` object, I removed `function` parameter in `FunctionInvokedContext` constructor. I leave it as it is in EventArgs though, to avoid changing public API.